app/alembic/versions/d04a8c11bec3_example.py

```diff
@@ -0,0 +1,35 @@
+"""example
+
+Revision ID: d04a8c11bec3
+Revises: 72534dacc9da
+Create Date: 2021-04-02 21:26:32.753544
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd04a8c11bec3'
+down_revision = '72534dacc9da'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+def upgrade():
+    op.create_table(
+        'example',
+        sa.Column('id', sa.Integer, primary_key=True),
+        sa.Column('name', sa.String(50), nullable=True),
+        sa.Column('description', sa.Unicode(200))
+    )
+    op.execute('INSERT INTO example (name, description) values (\'Austin\', \'A person who does things\')')
```

Comment:
Can we update this to
`op.execute('INSERT INTO example (name, description) values (\'Austin\', \'An awesome colleague, software engineer and golfer who will break 85. \')')`

Reply:
lol funny but not factual enough for this PR =P